src/main/resources/graphql/resource.graphqls

```diff
@@ -0,0 +1,24 @@
+"""
+Resources are all types of content present in a course. Each resource can be available or unavailable in a course.
+"""
+type Resource {
+    """
+    UUID of the resource.
+    """
+    resource_id: UUID!
+    """
+    A list of courses a resource is present in, including its availability
+    """
+    courses: [courseIdAvailabilityMap!]!
+}
+
+type courseIdAvailabilityMap {
```

Comment:
This needs to be PascalCase according to our guidelines.

Reply:
I would also not name this "Map" as it isn't a map. Maybe just CourseAvailabilityInformation or sth similar?

Reply:
As already discussed with Tim, this type will be removed in favor of just an array of courses